rust_src/src/data.rs

```diff
@@ -689,4 +691,82 @@ pub fn set_default(symbol: LispSymbolRef, value: LispObject) -> LispObject {
     value
 }
 
```

Comment:
I added that because of 
```rust
pub extern "C" fn map_obarray(
    obarray: LispObject,
    func: extern "C" fn(LispObject, LispObject),
    arg: LispObject,
)
```
Is there a way to cast `fn` as `extern "C" fn`?

Reply:
Ah, then you are doing the right thing as long as `map_obarray` is shared with C code.